src/Actions/Pending.ts

```diff
@@ -142,14 +178,14 @@ const hasPendingGeth = (
  * @param {number} minPrice (optional) Expected gasPrice.
  * @returns {Promise<boolean>} Transaction, if a pending transaction to this address exists.
  */
-const hasValidGasPrice = async (conf: any, transaction: any, minPrice?: any) => {
+const hasValidGasPrice = async (conf: Config, transaction: any, minPrice?: any) => {
```

Comment:
Maybe better to pass the current gasPrice from the network as parameters. Otherwise, hasValidGasPrice used from hasPendingPool will get the price many times in a very short time, we can safely assume this won't change.

Reply:
Replace `any`with types. We are going to enable `no-any` compiler flag 🔜